src/main/java/edu/flab/BeanConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package edu.flab;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class BeanConfiguration {
```

Comment:
```suggestion
public class CryptBeanConfiguration {
```

명시적인 이름이 좋을것 같아요 (더 좋은 이름이 있다면 그것으로..)

Reply:
넵 :) `CryptoBeanConfiguration` 으로 하려구요